app/controllers/api/v1/items/search_controller.rb

```diff
@@ -1,11 +1,65 @@
 class Api::V1::Items::SearchController < ApplicationController
 
   def index
-    render json: ItemSerializer.new(Item.where(request.query_parameters))
+    params_length = request.query_parameters.keys.count
+    render json: ItemSerializer.new(get_all_items) if params_length == 1
+    render json: ItemSerializer.new(Item.where(request.query_parameters)) if params_length > 1
```

Comment:
Multi-attribute searches that include partials need to be revised and refactored.